guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -10,6 +10,17 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-executing-a-remot
 . Select the *Job category* and the *Job template* you want to use, then click *Next*.
 . Select hosts on which you want to run the job.
 If you do not select any hosts, the job will run on all hosts you can see in the current context.
++
+[NOTE]
+====
+If you want to select a host group and all of its subgroups, it is not sufficient to select the host group as the job would only run on hosts directly in that group and not on hosts in subgroups.
+Instead, you must either select the host group and all of its subgroups or use this search query:
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+hostgroup_fullname ~ _My_Host_Group_/*
```

Comment:
```suggestion
hostgroup_fullname ~ _My_Host_Group_*
```

Tested on Foreman 3.8: Only if I drop the trailing slash I get all hosts within the parent host group and sub host groups.

Reply:
Confirmend and applied.